app/assets/stylesheets/media.scss

```diff
@@ -276,7 +276,7 @@
 
     h3 {
       align-self: center;
-      font-size: 0.937rem;
+      font-size: 1rem;
```

Comment:
I thought the display looked odd where the companion window section headers (`Transcript`, `Media Content`, etc.) were *smaller* than content underneath, such as the `Auto scroll` controls, so I bumped this font up a tiny bit. This looks closer to the design in #1806 to my eyes too.